structured/sql_api.proto

```diff
@@ -93,6 +93,6 @@ message SQLRequest {
 
 message SQLResponse {
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
-  // The result set.
+  // The result set for the last Cmd in the request.
```

Comment:
Well, this is better than what was there before. Is there a reason you don't want to return results for all of the commands in the request? Or is this just a stop-gap solution?